test/chaincodes/ledger/chaincode.js

```diff
@@ -0,0 +1,24 @@
+/*
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+*/
+'use strict';
+
+const {Contract} = require('fabric-contract-api');
+const {Ledger} = require('fabric-ledger');
+
+class LedgerChaincode extends Contract {
```

Comment:
Ledger*Chaincode*?

Reply:
LedgerAPIExample maybe?

Reply:
LedgerTestContract (since it's not really an example)